examples/tc_guidance_inference.py

```diff
@@ -37,7 +41,16 @@
     "cbottle-3d-moe-tc",
 )
 indices_where_tc = model.get_guidance_pixels(lons, lats)
-out, coords = model.sample(batch, guidance_pixels=indices_where_tc)
+out, coords = model.sample(
+    batch,
+    guidance_pixels=indices_where_tc,
+)
+
+if translate:
+    out = model.normalize_and_reorder(out)
+    batch["target"] = out
+    out, coords = model.translate(batch, dataset="icon")
+
```

Comment:
can you think of a way to hide this from the user? Would like to avoid adding the `normalize_and_reorder` public API, since it's not clear convention the src and dest are for "reordering". Also "and" i the name indicates the method does two things and should therefore not be a method.